modules/openapi-generator/src/main/resources/dart2/auth/http_basic_auth.mustache

```diff
@@ -1,16 +1,12 @@
-part of {{pubName}}.api;
-
+{{>header}}
+{{>part_of}}
 class HttpBasicAuth implements Authentication {
-
-  String _username;
-  String _password;
+  String username;
+  String password;
 
   @override
   void applyToParams(List<QueryParam> queryParams, Map<String, String> headerParams) {
-    String str = (_username == null ? "" : _username) + ":" + (_password == null ? "" : _password);
-    headerParams["Authorization"] = "Basic " + base64.encode(utf8.encode(str));
+    final str = (username == null ? '' : username) + ':' + (password == null ? '' : password);
```

Comment:
I think this line can get more optimized to this:
`final str = (username ??  '') + ':' + (password ?? '');`

Reply:
Yup :)

Reply:
Patched, awaiting CI.